monkey/monkey_island/cc/services/config.py

```diff
@@ -237,7 +237,7 @@
                                 + " Note that setting this value too high may result in the monkey propagating too far"
                         },
                         "subnet_scan_list": {
-                            "title": "Scan IP/subnet list",
+                            "title": "Scan IP/subnet/domain names list",
```

Comment:
better to call it hostnames, domain names can also mean saying (for example) contoso.com which might imply the whole domain.